pilot/pkg/model/push_context.go

```diff
@@ -95,6 +96,9 @@ type PushContext struct {
 	// ServiceAccounts contains a map of hostname and port to service accounts.
 	ServiceAccounts map[host.Name]map[int][]string `json:"-"`
 
+	// AuthPolicies contains a map of hostname and port to authentication policy
+	AuthPolicies map[host.Name]map[int]*Config `json:"-"`
```

Comment:
Could you rename to `AuthnPolicies` to make it explicit it's for authentication? Otherwise it looks very confusing with the current `AuthzPolicies`.

Reply:
done